modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="supporting-encapsulation_{context}"]
+= Supporting encapsulation
+
+As an OpenShift support engineer or developer, you want to inspect Kubernetes-specific mountpoints.
+
+== Inspecting the encapsulated namespaces
```

Comment:
Since this is a module, we don't use second level headings, see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-procedures. Perhaps the section could just be titled, for example, "Inspecting encapsulated namespaces"?

Reply:
Done. We need them as highlighted headings.